### DISCUSSION TITLE
Outage window for Teleport Cloud upgrades - 17 hours?

### DISCUSSION THREAD

**Discussion by gord-lyr** [gravitational/teleport#23991] [General]
The scheduled maintenance notification we received indicated that there would be an outage during the maintenance, and maintenance has an estimated duration of 17 hours.

I'm assuming this is just the window for all upgrades - but how much downtime should we expect? 10 minutes? 30 minutes?

Thanks,
Gord T.

**Comment by webvictim** [comment]
From the Teleport Cloud team: "We are doing the upgrades respecting the tenant's selected window, and the outage for any given tenant is usually below 30 minutes."
